### ISSUE TITLE
So what do we do about these builds failing?

### ISSUE THREAD

**Issue by awebeer256** [ublue-os/nvidia#116]
I've been following uBlue for about two weeks. Do we just wait for Nvidia and/or Fedora to fix it?

(For the lazy, the Fedora 38 builds are failing on line 40 of `build.sh`: `akmods --force --kernels "${KERNEL_VERSION}" --kmod "${NVIDIA_PACKAGE_NAME}"`. Partial output:
```
Checking kmods exist for 6.3.5-200.fc38.x86_64 [  OK  ]
Building and installing nvidia-kmod [FAILED]
```
)

**Comment by castrojo** [comment]
We need to either wait for it to be fixed or someone needs to PR a fix into the akmods repo.

**Comment by akdev1l** [comment]
The first step would be to reproduce the build failure on Fedora Workstation - if that's possible then the issue belongs upstream

**Comment by awebeer256** [comment]
> We need to either wait for it to be fixed or someone needs to PR a fix into the akmods repo.

Do you mean the uBlue akmods repo? Because my fork doesn't use that at all, and it still started failing at the same time.

**Comment by castrojo** [comment]
Yeah there's a bunch of failures across the board with F38 in akmods, main, and nvidia.

**Comment by antuan1996** [comment]
Is it possible to disable "v4l2loopback" in ublue-os/akmods repository temporary to fix the build?

**Comment by castrojo** [comment]
I'm not familiar with how this repo works but if someone has a fix to disable it to unjam the rest I'll review it!

**Comment by awebeer256** [comment]
I'll repeat myself: my fork of this repo started failing at the same time as this repo did, and mine doesn't use the akmods repo at all. So that can't be the source of the problem.

**Comment by castrojo** [comment]
Sorry I should have finished my thought (was on mobile), there are multiple
rpmfusion related failures right now which are causing all the builds to
fail so each of them will need investigation.

**Comment by EyeCantCU** [comment]
Yup. I spent a lot of time looking into the cause. Just waiting on rpmfusion related things to get worked out

**Comment by antuan1996** [comment]
In awebeer256/ublue-nvidia build logs:
```
2023-06-08T03:27:54.1615418Z Installing: akmod-nvidia-3:530.41.03-1.fc38.x86_64 (rpmfusion-nonfree)
2023-06-08T03:27:54.1730098Z Building /usr/src/akmods/nvidia-kmod-530.41.03-1.fc38.src.rpm for kernel 6.3.5-200.fc38.x86_64
2023-06-08T03:27:54.1777532Z ERROR: Not to be used as root; start as user or 'akmodsbuild' instead.
```

In blue-os/akmods logs:
```
2023-06-12T13:04:28.4023326Z Installing: akmod-v4l2loopback-0.12.7^20230503g2c9b670-1.fc38.x86_64 (rpmfusion-free-updates)
2023-06-12T13:04:28.4149248Z Building /usr/src/akmods/v4l2loopback-kmod-0.12.7^20230503g2c9b670-1.fc38.src.rpm for kernel 6.3.6-200.fc38.x86_64
2023-06-12T13:04:28.4201899Z ERROR: Not to be used as root; start as user or 'akmodsbuild' instead.
```

**Comment by awebeer256** [comment]
@antuan1996 What's your point? That's been happening all along, in both successful and failed builds.
